tests/spec/run.rs

```diff
@@ -491,7 +491,7 @@ fn try_spec(name: &str) -> Result<(), Error> {
         for err in errors {
             write!(out, "{}", err).expect("Error formatting errors");
         }
-        panic!(out);
+        panic!("{}", out);
```

Comment:
Maybe you could use Rust's newly stabilized `panic_any` here: https://doc.rust-lang.org/std/panic/fn.panic_any.html

Reply:
Sure, let me have a look

Reply:
`out` being a `String`, what benefit do you see using `panic_any` ?

Reply:
ah then it probably won't make sense. please go forward. :)